packages/block-editor/src/components/inserter/preview-panel.js

```diff
@@ -8,6 +8,7 @@ import {
 	getBlockType,
 } from '@wordpress/blocks';
 import { __ } from '@wordpress/i18n';
+import { select } from '@wordpress/data';
```

Comment:
the usage of the global 'select' or 'dispatch' in Gutenberg should be limited as much as possible because they are singletons and not aware of the parent registries (nested contexts). They  are also not reactive. So prefer `useSelect` and `useDispatch` instead.